FinalTask/Pages/AccountCreationPage.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace FinalTask.Pages
+{
+    public class AccountCreationPage: BasePage
+    {
+        private By maleGenderLocator = By.XPath("//imput[@id = 'id_gender1']");
+        private By femaleGenderLocator = By.XPath("//input[@id = 'id_gender2']");
+        private By firstNameLocator = By.XPath("//input[@id = 'customer_firstname']");
+        private By lastNameLocator = By.XPath("//input[@id = 'customer_lastname']");
+        private By passwordLocator = By.XPath("//input[@id = 'passwd']");
+
+        private By addressFirstNameLocator = By.XPath("//input[@id = 'firstname']");
+        private By addressLastnameLocator = By.XPath("//input[@id = 'lastname']");
+        private By addressLocator = By.XPath("//input[@id = 'address1']");
+        private By cityLocator = By.XPath("//input[@id = 'city']");
+        private By stateLocator = By.XPath("//select[@id = 'id_state']");
+        private By zipCodeLocator = By.XPath("//input[@id = 'postcode']");
+        private By mobilePhoneLocator = By.XPath("//input[@id = 'phone_mobile']");
+        private By registerButtonLocator = By.XPath("//button[@id = 'submitAccount']");
+
+
+        public AccountCreationPage(IWebDriver driver) : base(driver)
+        {
+        }
+
+        public MyAccountPage CreateAccount(Gender gender, string firstName, string lastName, string password, string address, string city, string state, string zip, string mobilePhone)
```

Comment:
There are too many method parameters to pass them each time when you need to create account. I would create an Account class to keep all data about account record there and just pass a instance of this type into CreateAccount(Account account) method.